tests/test_connections.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from unittest.mock import patch
+from app import create_app
+from app.main.data import get_remote_data
+
+
+@pytest.fixture
+def app():
+    app = create_app()
+    yield app
+
+
+@pytest.fixture
+def client(app):
+    return app.test_client()
```

Comment:
Best practice to use a context manager when using a test client. This example is just from the API.

```
@pytest.fixture()
def flask_test_client():
    """
    Creates a test client.
    :return: A flask test client.
    """
    with create_app().test_client() as test_client:
        yield test_client
```

Reply:
Think we should also probably pull this out to a [`conftest.py`](https://docs.pytest.org/en/6.2.x/fixture.html#conftest-py-sharing-fixtures-across-multiple-files) file as it can be used across `test_connections` and `test_routes` and any new test modules

Reply:
I've pulled it out into a conftest file and updated the test scripts accordingly.